### PR TITLE
Jetpack Visual Refresh: Footer

### PR DESCRIPTION
**Pull request by nateweller**
## Proposed changes:

* Updates the `JetpackFooter` component from the `jetpack-components` package to include support for a `menu` prop to render a navigation menu beside the Jetpack logo in the footer. (https://github.com/Automattic/jetpack/pull/30279/commits/5a2cf7b6d6d2f9c90c21e8a662e7b8d49ffced30)
* Updates the Jetpack plugin to use the new `JetpackFooter` menu:
  * Updates React footer component to use the package component for the menu. (https://github.com/Automattic/jetpack/pull/30279/commits/c4f60bd6c26664652e0d779aec4d7d8a93ba835b)
  * Updates the PHP footer and related `.jp-footer` styles to apply the same styles from the package. (https://github.com/Automattic/jetpack/pull/30279/commits/c4f60bd6c26664652e0d779aec4d7d8a93ba835b)
  * Updates the "Recommendations" bottom navigation to have styles independent of the footer. (https://github.com/Automattic/jetpack/pull/30279/commits/41aaaf6f26d944e3c572d0138034b0abf921b3e1)

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

https://github.com/orgs/Automattic/projects/587/views/15?pane=issue&itemId=26242471

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Review the footer on React screens (at a glance page, settings screens).
* Review the footer on static screens (about, modules).
* Review the bottom links from the "Recommendations" tab.

## Screenshots

Mobile

Screenshot 2023-04-26 at 3 15 38 PM

Tablet

Screenshot 2023-04-26 at 3 15 28 PM

Desktop

Screenshot 2023-04-26 at 3 15 18 PM

Recommendations 

Screenshot 2023-04-26 at 3 43 34 PM

### CODE REVIEW

**Comment by jeffgolenski** [comment]
@nateweller outstanding. send it!

File: projects/plugins/jetpack/_inc/client/components/footer/style.scss [hunk 1]
```diff
@@ -1,81 +1,106 @@
...
+	}
 }
 
 .jp-footer__links {
-	border-top: 1px lighten( $gray, 25% ) solid;
-	border-bottom: 1px lighten( $gray, 25% ) solid;
-	margin-top: 0;
-	margin-bottom: rem( 16px );
+	margin: rem( 24px ) auto;
+	display: flex;
```
Comment (jeffgolenski) [inline projects/plugins/jetpack/_inc/client/components/footer/style.scss]: Nice flexin'

**Review by jeffgolenski — DISMISSED** [review]

**Review by dhasilva — CHANGES_REQUESTED** [review]
Some pages are displaying a SVG tag in the Privacy link:

File: projects/plugins/jetpack/_inc/client/recommendations/style.scss [hunk 2]
```diff
@@ -47,3 +47,51 @@
...
+	
+		&:hover {
+			text-decoration-thickness: var( --jp-underline-thickness );
+		}
+
+		&:hover:focus {
+			text-decoration: none;
+		}
+	}
+}
```
Comment (dhasilva) [inline projects/plugins/jetpack/_inc/client/recommendations/style.scss]:
```suggestion
}

```

**Comment by nateweller** [comment]
@dhasilva Thanks for your prompt review! Fixed it up :+1:

**Review by dhasilva — APPROVED** [review]
LGTM. Nice!

**Comment by zinigor** [comment]
Cherry picked in https://github.com/Automattic/jetpack/commit/e63eaff87c8f8a860407f75774ba01ea240e636a.

**Pull request merged**